addons/uh60_doorguns/functions/fnc_setup.sqf

```diff
@@ -4,6 +4,8 @@
 
 params ["_vehicle"];
 
+if (!vtx_uh60m_enabled_m134_recoil) exitWith {false};
```

Comment:
This would disable the whole doorguns module - is this what we want?

Reply:
Nah, lets just close this. 

Think it was just one guys graphics settings messing it up